core/systems/project/item_templates.ex

```diff
@@ -1,3 +1,8 @@
 defmodule Systems.Project.ItemTemplates do
-  use Core.Enums.Base, {:project_item_templates, [:benchmark_challenge, :data_donation]}
+  if Application.compile_env(:core, :leaderboard_enabled) do
```

Comment:
Does this work together with running `mix gettext.extract --merge` for generating label translations?